app/src/test/java/org/mozilla/tv/firefox/helpers/TestTurboMode.kt

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.helpers
+
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.MutableLiveData
+import org.mozilla.tv.firefox.utils.TurboMode
+
+/**
+ * An implementation of [TurboMode] that does not persist state between
+ * instances.
+ */
+class TestTurboMode(private var isEnabled: Boolean) : TurboMode {
```

Comment:
This appears to be unused. If so, do we still need the interface and prod implementation distinction?

Reply:
I swear this had a point when I wrote it....  Think I removed the class that required it.  I'll pull it out.